candidates/views.py

```diff
@@ -953,7 +953,7 @@ def get_context_data(self, **kwargs):
             for req_progress in progress_item['requirements']:
                 requirement = req_progress['requirement']
                 # Exclude progresses with the wrong requirement
-                if self.display_req and self.display_req != requirement:
+                if not requirement or self.display_req and self.display_req != requirement:
```

Comment:
Use the following syntax (prioritize easy to read, even though logic may be is the same if you were thinking of that)

`if (not requirement) or (self.display_req and self.display_req != requirement):`

Reply:
done